jdaviz/configs/imviz/plugins/footprints/footprints.py

```diff
@@ -29,7 +29,7 @@ class Footprints(PluginTemplateMixin, ViewerSelectMixin, HasFileImportSelect):
     :ref:`public plugin API <plugin-apis>`:
 
     * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.show`
-    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.open_in_tray`
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.open_in_tray`

```

Comment:
what was changed here?

Reply:
There isn't any modification to the text and no additional whitespace so I don't see why it is showing up in the diff